src/fides/api/alembic/migrations/versions/7641ea685ea4_add_google_cloud_sql_mysql_to_.py

```diff
@@ -0,0 +1,96 @@
+"""add google_cloud_sql_mysql to connectiontype
+
+Revision ID: 7641ea685ea4
+Revises: cb344673f633
+Create Date: 2024-06-18 16:17:15.012408
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '7641ea685ea4'
+down_revision = 'cb344673f633'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add 'google_cloud_sql_mysql' to ConnectionType enum
+    op.execute("ALTER TYPE connectiontype RENAME TO connectiontype_old")
+    op.execute(
+        """
+        CREATE TYPE connectiontype AS ENUM (
```

Comment:
Looks like the `s3` connection type was added a few days ago, we'll need to update this one more time